src/www/views/servers_view/server_connection_indicator/index.ts

```diff
@@ -67,13 +67,19 @@ export class ServerConnectionIndicator extends LitElement {
       .circle {
         display: inline-block;
 
+        backface-visibility: hidden;
```

Comment:
should probably add a comment here

Reply:
I think we can add some hints, at least it doesn't seem to be a common css property to me.